src/Compilers/CSharp/Test/Syntax/Parsing/PartialEventsAndConstructorsParsingTests.cs

```diff
@@ -0,0 +1,1552 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests;
+
+public sealed class PartialEventsAndConstructorsParsingTests(ITestOutputHelper output) : ParsingTests(output)
+{
+    private sealed class FeatureLangVersions()
+        : CombinatorialValuesAttribute(LanguageVersionFacts.CSharpNext, LanguageVersion.Preview);
+
+    private sealed class LastThreeLangVersions()
```

Comment:
don't love this name, unless we are planning on updating it when we continue to add new lang versions.  Consider CSharp13_Next_Preview

Reply:
Good point, I will rename this, thanks.

Reply:
I will go with "CSharp14" instead of "Next" since this would likely not get renamed once Next moves.